src/api/walker.ts

```diff
@@ -10,7 +10,8 @@ import * as resolveSymlink from "./functions/resolve-symlink";
 import * as invokeCallback from "./functions/invoke-callback";
 import * as walkDirectory from "./functions/walk-directory";
 import { Queue } from "./queue";
-import { Dirent } from "fs";
+import type { Dirent } from "fs";
+import * as nativeFs from "fs";
```

Comment:
just a style question probably for @thecodrr, but I wonder if these two lines should be combined into a single line